.circleci/config.yml

```diff
@@ -23,8 +23,13 @@ jobs:
           name: 'Unit Tests'
           command: |
             source /usr/local/share/virtualenvs/tap-sftp/bin/activate
-            pip install nose
-            nosetests tests/unittests/
+            pip install nose coverage
```

Comment:
Please update the tap-tester image to the latest `stitch-tap-tester`

Reply:
@kspeer825 When we are updating image we are getting `ModuleNotFoundError: No module named 'paramiko'` error. Please find the error [reference](https://app.circleci.com/pipelines/github/singer-io/tap-sftp/819/workflows/1914ce56-14c8-48e0-85ad-f247e4b4661f/jobs/986?invite=true#step-107-27)

Reply:
That dependency will need to be imported directly in the circleci config. The preferred method for this is to use a `test` group under extras_require in the setup.py. Then this will have to be installed via pip under the integration test step in the workflow definition. This was one of the breaking changes to tap-tester that resulted in the image actually - removing tap dependencies from the test framework repo.